packages/web/.eslintrc.js

```diff
@@ -18,6 +18,10 @@ module.exports = {
         ArrayPattern: { multiline: true },
       },
     ],
+    "@typescript-eslint/no-unused-vars": "error",
```

Comment:
The project is still in development and has unused variables declared.
Why not change it to `warn` and build the project?
I'll work on changing the `typescript-eslint/no-unused-vars` option to `error`.